src/main/java/com/proudmuslim/andromeda/mixin/PlayerEntityMixin.java

```diff
@@ -13,18 +10,17 @@
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
-// Mixins HAVE to be written in java due to constraints in the mixin system.
+/** Mixins HAVE to be written in java due to constraints in the Mixin system. */
 @Mixin(PlayerEntity.class)
 public abstract class PlayerEntityMixin extends LivingEntity {
 
-    protected PlayerEntityMixin(EntityType<? extends LivingEntity> type, World world) {
+    protected PlayerEntityMixin(final EntityType<? extends LivingEntity> type, final World world) {
```

Comment:
que?

Reply:
final = speed

Reply:
also i think it's in my style settings to do that and i ran an automatic thing